CMakeLists.txt

```diff
@@ -60,7 +67,8 @@ target_include_directories(${PROJECT_NAME} PUBLIC
 
 target_include_directories(${PROJECT_NAME}_exported PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-  $<INSTALL_INTERFACE:include>)
+  $<INSTALL_INTERFACE:include>
+  $ENV{OpenVINO_INCLUDE_DIRS})
```

Comment:
Why appending `OpenVINO_INCLUDE_DIRS` variable? Since it already link to the `openvino::runtime`, it seems that OpenVino headers should already be included, try removing it and see if it works the same.

```suggestion
```